src/get-authentication-context.js

```diff
@@ -0,0 +1,6 @@
+import Cookies from 'js-cookie'
+import isAuthenticatedWithContext from './is-authenticated-with-context'
+
+export default function () {
+  return isAuthenticatedWithContext() && JSON.parse(Cookies.get('lp_auth'))['options']
```

Comment:
i'm all for code sharing, but in this situation its prob not the best idea since you end up repeating some I/O and parsing activity. In this case, its pretty small, but in general we want to minimize those things.